auth/src/main/java/com/firebase/ui/auth/provider/FacebookProvider.java

```diff
@@ -55,6 +55,8 @@
     private IdpCallback mCallbackObject;
 
     public FacebookProvider(Context appContext, IdpConfig idpConfig) {
+        appContext = appContext.getApplicationContext();
```

Comment:
Do we want to do this or only call `getApplicationContext` when we are passing it into the Facebook initializer?